package.nls.zh-tw.json

```diff
@@ -20,5 +20,39 @@
     "Common.notNow": "先不要",
     "Common.gotIt": "懂了！",
     "downloading.file": "正在下載 {0}...",
-    "downloading.file.progress": "目前 {0}{1}，總共 {2} KB ({3}%)"
+    "downloading.file.progress": "目前 {0}{1}，總共 {2} KB ({3}%)",
+    "DataScience.restartingKernelStatus": "重新啟動 Jupyter 核心中",
+    "DataScience.collapseVariableExplorerLabel": "變數",
+    "DataScience.notebookExport": "匯出",
+    "DataScience.notebookExportAs.shorttitle": "匯出",
+    "DataScience.exportButtonTitle": "匯出",
+    "DataScience.exportAs": "匯出為",
+    "DataScience.notebookExportAs": "匯出為",
+    "DataScience.exportAsQuickPickPlaceholder": "匯出為...",
+    "DataScience.runByLine": "逐行執行 (F10)",
+    "DataScience.debugCellCommandTitle": "偵錯儲存格",
+    "DataScience.exportPythonQuickPickLabel": "Python 指令碼,
```

Comment:
@ted1030 Looks like there might be a missing " here on this line.

Reply:
Fixed.